app/src/main/java/com/example/atheneum/activities/FirebaseUIAuthActivity.java

```diff
@@ -115,7 +115,7 @@ private void generateSignInPage() {
                 AuthUI.getInstance()
                         .createSignInIntentBuilder()
                         .setAvailableProviders(providers)
-//                                        .setIsSmartLockEnabled(false) // Disabled for testing, used for password-less login
+                        .setIsSmartLockEnabled(false) // Used for password-less login, disabled to make UI tests easy

```

Comment:
This should be fine, but has problems if the user was logged in before. In my case it was still logged in when the app opened at the test failed immediately. 
For anyone running this just remember to log out.

Reply:
I made a new change that checks the user's auth status and their connectivity status. The test throws an assert if either the user is logged or they are offline. This is to ensure that the tests start in a reliable state.